test/sanity/validate-modules/ignore.txt

```diff
@@ -88,6 +88,7 @@ lib/ansible/modules/cloud/amazon/sts_assume_role.py E317
 lib/ansible/modules/cloud/atomic/atomic_container.py E317
 lib/ansible/modules/cloud/azure/azure_rm_dnszone_facts.py E325
 lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py E325
+lib/ansible/modules/cloud/azure/azure_rm_sqldatabase.py E309
```

Comment:
This ignore should not be added. The test will pass once merged.

Reply:
ok, so we should ignore failing test when merging?

Reply:
When the only failure on a PR is a false positive error for version added, then yes, it can be merged. We just have to make sure that is the only error. You can't tell from GitHub, you'll have to look at the Shippable test matrix to verify.

However, in this case I'm not sure the error is a false positive. We're enabling a feature that wasn't present on this module before, so the version added value should be 2.8 -- but only for this module.

Reply:
yes, I am not sure what to do, as tags is generic documentation fragment, so I can't add version_added, unless I define tags as a custom option, which doesn't seem right.
so I think better solution will be to not have version_added.